test/TOOLS/do_tests

```diff
@@ -10,7 +10,8 @@ if [ "$1" = "smt_tests" ]
   then export ARGS="$ARGS --method smt"
 fi
 
-export TLAPM="$BASEDIR/../tlapm -I $ARGS"
+export TLAPM="$BASEDIR/bin/tlapm"
```

Comment:
This leads to tlapm being called from the test/bin directory. Even when I put the tlapm binary there, the missing -I flag fails all tests that extend TLAPS. Is this the way we want it?

Reply:
In fact it should be written somewhere, but the correct setup is to use the installer to install the whole system into the test directory:
```
tlaps-1.4.5-*-inst -d /home/.../test
```
This means that we test not only the program itself, but also the installation process.